meveo-ejb/pom.xml

```diff
@@ -0,0 +1,87 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
I thought we agreed to not rename meveo-ejb, and keep meveo-admin-ejbs

Reply:
I have to combined all ejb module as a sing
le jar . So, satisfy this case
I have created a ejb integraion module and  its artifact name is "meveo-ejb"

If you want to change this artifact name? Please give me your artifact name
other than "meveo-admin-ejbs" .